src/index.html

```diff
@@ -1,20 +1,61 @@
 <html>
-  <head> </head>
+  <head>
+    <style type="text/css">
+      #animation-container {
+        background: red;
+      }
+      #player-one {
+        background: green;
+      }
+      .arm-l path {
+        fill: purple;
+      }
+
+      .body path {
+        fill: red;
+      }
+
+      .leg-l path {
+        fill: orange;
+      }
+
+      .leg-r path {
+        fill: orange;
+      }
+
+      .arm-r path {
+        fill: turquoise;
+        stroke-width: 8;
+        stroke: pink;
+        stroke-linecape: round;
+      }
+    </style>
+  </head>
   <body>
     <div
       class="player-container"
       style="width: 500px; height: 500px; margin: auto; justify-content: center"
     >
       <lottie-player
-        src="https://assets3.lottiefiles.com/packages/lf20_UJNc2t.json"
+        src="https://lottie.host/4458c0b5-4d8d-43f3-8d42-0739a212a523/L4yrEGvBsO.json"
         autoplay
         controls
         loop
         disableCheck
         mode="normal"
-        viewBoxSize="0 0 50 600"
```

Comment:
@samuelOsborne 
Hi Samuel!
i think this property is needed.. there's a reason to be removed?
Thanks in advance 👋

Reply:
hi @PsHye it was just removed from the example test code we use, not the actual player

Cheers!